RefugeeLand.Core.Api/Migrations/StorageBrokerModelSnapshot.cs

```diff
@@ -1,12 +1,9 @@
-﻿// -------------------------------------------------------
-// Copyright (c) Coalition of the Good-Hearted Engineers
-// FREE TO USE TO DELIVER HUMANITARIAN AID, HOPE AND LOVE
-// -------------------------------------------------------
-
-// <auto-generated />
+﻿// <auto-generated />
```

Comment:
Missing header

Reply:
You don't need header file when you see
`// <auto-generated />`